lib/constants.dart

```diff
@@ -2,4 +2,5 @@ class AppConstsnts {
   static const String endPoint = "https://demo.appwrite.io/v1";
   static const String project = "606d5bc9de604";
   static const String collection = "606d5bd9626ae";
+  static const String databaseID = "";
```

Comment:
Should be **"default"** here?

Reply:
I didn't know the database ID for tutorial. Lemme apply this in a few